app/assets/stylesheets/bootstrap_custom.scss

```diff
@@ -46,8 +46,8 @@ $link-hover-decoration: underline !default;
 //
 //## Font, line-height, and color for body text, headings, and more.
 
-$font-family-sans-serif:  "Helvetica Neue", Helvetica, Arial, sans-serif !default;
-$font-family-serif:       Georgia, "Times New Roman", Times, serif !default;
+$font-family-sans-serif:  Roboto;
```

Comment:
You want to provide a fallback here pretty please

Reply:
I have re-added the sans-serif fonts as the fallback.